.gitignore

```diff
@@ -4,3 +4,6 @@ src/gui/runtime/doc/tags
 third-party/msgpack*
 compile_commands.json
 !.github
+
+# Qt Creator project files
+CMakeLists.txt.user
```

Comment:
Let's remove this, better to have IDE-specific files in the user's `.gitignore`.

Reply:
I think, given that the project is using C++ stack and has a Qt GUI application component, it is reasonable to exclude this file, because there is a high chance a developer might be using Qt Creator.